plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -3747,6 +3756,66 @@ private void testMultipleWriters(Session session, HiveStorageFormat storageForma
         }
     }
 
+    @Test
+    public void testMultipleWritersWhenTaskScaleWritersIsEnabled()
+    {
+        long workers = (long) computeScalar("SELECT count(*) FROM system.runtime.nodes");
+        int defaultMinScaledWriterCount = 8;
```

Comment:
I would avoid hard coding it here. It is not obvious this value has to be changed when the default changes.